### PR TITLE
[pcieutil] Remove the warning message and change the config file location

### PR DESCRIPTION
**Pull request by sujinmkang**
**- What I did**
Fixes https://github.com/Azure/sonic-buildimage/issues/6437

**- How I did it**
Changed the configuration file path from plugin to platform directory 

**- How to verify it**
Verify on the device

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: pcieutil/main.py [hunk 1]
```diff
@@ -41,22 +41,17 @@ def print_result(name, result):
...
-    global platform_plugins_path
+    global platform_path
 
     # Load platform module from source
     try:
         platform_path, _ = device_info.get_paths_to_platform_and_hwsku_dirs()
-        platform_plugins_path = os.path.join(platform_path, "plugins")
-        sys.path.append(os.path.abspath(platform_plugins_path))
-        from pcieutil import PcieUtil
+        sys.path.append(os.path.abspath(platform_path))
```
Comment (jleveque) [inline pcieutil/main.py:49]: Why do we need to append `platform_path` to `sys.path`? We're not loading Python modules from here, correct? Only the pcie.yaml file resides there, right?

Comment (sujinmkang) [reply pcieutil/main.py:49]: correct.

File: pcieutil/main.py [hunk 2]
```diff
@@ -131,7 +125,7 @@ def check():
 def generate():
     '''Generate config file with current pci device'''
     platform_pcieutil.dump_conf_yaml()
-    click.echo("Generate config file pcie.yaml under path %s" % platform_plugins_path)
+    click.echo("Generate config file pcie.yaml under path %s" % platform_path)
```
Comment (jleveque) [inline pcieutil/main.py:128]: Suggest to change to `click.echo("Generated config file '{}/pcie.yaml'".format(platform_path))`

**Review by jleveque — CHANGES_REQUESTED** [review]

File: pcieutil/main.py [hunk 3]
```diff
@@ -131,7 +125,7 @@ def check():
 def generate():
     '''Generate config file with current pci device'''
     platform_pcieutil.dump_conf_yaml()
-    click.echo("Generate config file pcie.yaml under path %s" % platform_plugins_path)
+    click.echo("Generate config file '{}/pcie.yaml'".format(platform_path))
```
Comment (jleveque) [inline pcieutil/main.py:128]: s/Generate/Generated/

**Review by jleveque — CHANGES_REQUESTED** [review]

**Review by jleveque — APPROVED** [review]

**Comment by sujinmkang** [comment]
retest this please

**Comment by dprital** [comment]
@sujinmkang  - Can you please merge it as well to 202012 ?

**Comment by dprital** [comment]
@daall - Can you please merge to 202012 please ?

**Pull request merged**